xml/docu_styleguide.language.xml

```diff
@@ -383,6 +383,11 @@
    documents. Do not use humor. Be honest and avoid absolutes and
    exaggerations, but focus on positive aspects.
   </para>
+  <para>
+   Do not make promises for the future.
+   Document only existing features and features that will be finished before
+   the document is first published.
+  </para>
```

Comment:
* If PM pressures us, we can ask them to publish their roadmap documents outside of a documentation context and link there
* Technology previews are usually not documented, unless we are asked to by PM